bitnami/kubeapps/templates/ingress.yaml

```
@@ -9,8 +9,16 @@ metadata:
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" . ) | nindent 4 }}
     {{- end }}
   annotations:
-    {{- if .Values.ingress.certManager }}
-    kubernetes.io/tls-acme: "true"
+    {{- with .Values.ingress.certManager }}
+      {{- with .clusterIssuer }}
+    cert-manager.io/cluster-issuer: {{ . }}
+      {{- end }}
+      {{- with .httpIngressClass }}
+    acme.cert-manager.io/http01-ingress-class: {{ . }}
+      {{- end }}
+      {{- with .commonName }}
+    cert-manager.io/common-name: {{ . }}
+      {{- end }}
     {{- end }}
```

To keep the same style as the other charts, could you please use the following syntax instead?
```suggestion
    {{- if .Values.ingress.certManager.clusterIssuer }}
    cert-manager.io/cluster-issuer: {{ .Values.ingress.certManager.clusterIssuer }}
    {{- end }}
    {{- if .httpIngressClass }}
    acme.cert-manager.io/http01-ingress-class: {{ .Values.ingress.certManager.httpIngressClass }}
    {{- end }}
    {{- if .Values.ingress.certManager.commonName }}
    cert-manager.io/common-name: {{ .Values.ingress.certManager.commonName }}
    {{- end }}
```

---

Hello
Done. I had to leave a top-level `{{- if .Values.ingress.certManager }}` condition as this value may missing.

---

Nevermind my last comment, I've noticed that the CI failed because of that, so I've set a default value.